src/Sarif.Sdk.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29709.97
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31728.443
```

Comment:
Question, this is version 2022 right, do we want to upgrade, should we revert this change?

Reply:
it does not matter if u are using VS2019 or VS2022.

The nuget change is just to show when we open the sln.